### ISSUE TITLE
Could not get response

### ISSUE THREAD

**Issue by krisnguyen95** [fent/node-ytdl-core#586]
My server has been running the ytdl-core library for some time but now getInfo doesn't return a response anymore. I have a log in the library but only the request, not any response. Help me please
<img width="769" alt="Screen Shot 2020-03-04 at 10 27 01 PM" src="https://user-images.githubusercontent.com/58581971/75895048-acaeca00-5e67-11ea-8ee2-f28e8ff568a7.png">

**Comment by TimeForANinja** [comment]
looks like you're not on latest version 🤔

**Comment by rpradeepam** [comment]
I am seeing a similar behavior. Getting responses properly, but not getting errors where I'm expecting to get 429, 403, etc.
Could it be because of this?
<img width="469" alt="image" src="https://user-images.githubusercontent.com/16433794/80939020-c84b3800-8df8-11ea-9851-a057c5200c40.png">
When using a callback, if the `getBasicInfo` function thrown an error, it is not passed to the callback

**Comment by fent** [comment]
does it say something like unhandled promise error thrown or uncaught error?

**Comment by rpradeepam** [comment]
I hadn't been able to capture logs consistently. I have moved to using promises now anyway.

I had my code setup to check for 429 errors and handle them by getting a new IP, which was working fine. 
Over the past few months I had noticed that occasionally the `getInfo` method would stop returning data. My requests would just time out. 
Getting a new IP seemed to fix the problem, leading me to believe that I was getting 429s but not able to capture them.
